apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-hosts/facility-hosts.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { FacilitiesManagerService, Facility, Host } from '@perun-web-apps/perun/openapi';
+import {
+  TABLE_FACILITY_HOSTS_LIST,
+  TableConfigService
+} from '@perun-web-apps/config/table-config';
+import { ActivatedRoute } from '@angular/router';
+import { SelectionModel } from '@angular/cdk/collections';
+import { PageEvent } from '@angular/material/paginator';
+
+@Component({
+  selector: 'perun-web-apps-facility-hosts',
+  templateUrl: './facility-hosts.component.html',
+  styleUrls: ['./facility-hosts.component.scss']
+})
+export class FacilityHostsComponent implements OnInit {
+
+  constructor(private dialog: MatDialog,
+              private facilitiesManager: FacilitiesManagerService,
+              private tableConfigService: TableConfigService,
+              private route: ActivatedRoute) {
+
+  }
+
+  @Input()
```

Comment:
This shouldn't be here at all. How to use Input() and Output() you can see here https://angular.io/guide/template-syntax#input-and-output-properties